BrowserKit/Sources/ComponentLibrary/Headers/NavigationHeaderView.swift

```diff
@@ -2,17 +2,21 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
-import Foundation
 import UIKit
 import Common
-import Shared
-import ComponentLibrary
 
-class TrackingProtectionNavigationHeaderView: UIView {
+public protocol NavigationHeaderViewActionsHandler: AnyObject {
+    func backToMainView()
+    func dismissMenu()
+}
+
+public final class NavigationHeaderView: UIView {
```

Comment:
🔥 Yay more components in the library! If we're planning to add this, could we also add this in the sample component library as well as [documentation](https://mozilla-mobile.github.io/firefox-ios/documentation/componentlibrary/) similar to our other views? This could be done in a separate PR.

Reply:
hey @cyndichin may you create a ticket, please, for that? I will appreciate that! Thank you!

Reply:
thanks @dicarobinho~ Created here: https://mozilla-hub.atlassian.net/browse/FXIOS-10083